From: Ferdinand Bachmann <ferdinand.bachmann@yrlf.at>
Date: Tue, 30 Jan 2024 12:44:18 +0100
Subject: Fix USB scanners not working with RestrictAddressFamilies

colord-sane scanner drivers using libusb can't initialize properly with
RestrictAddressFamilies set to AF_UNIX. Remove that line to ensure those
can work properly.

This also avoids a crash in HPLIP due to unchecked calls to libusb_init().

Fixes #165

(cherry picked from commit 9283abd9c00468edb94d2a06d6fa3681cae2700d)

Origin: upstream after 1.4.7
---
 data/colord.service.in | 1 -
 1 file changed, 1 deletion(-)

diff --git a/data/colord.service.in b/data/colord.service.in
index c358dc4..45ec581 100644
--- a/data/colord.service.in
+++ b/data/colord.service.in
@@ -15,7 +15,6 @@ ProtectKernelModules=true
 ProtectKernelLogs=true
 ProtectControlGroups=true
 RestrictRealtime=true
-RestrictAddressFamilies=AF_UNIX
 
 ConfigurationDirectory=colord
 StateDirectory=colord
